forc/src/utils/plugin_descriptions.rs

```diff
@@ -0,0 +1,27 @@
+pub enum Plugin {
+    ForcFormat,
+    ForcLsp,
+    ForcExplore,
+    None,
+}
+
+pub fn plugin_name_to_enum(n: &str) -> Plugin {
```

Comment:
Would obviously need an entire list of all `forc` plugins and their respective descriptions